### PR TITLE
adding bitrig cargo snapshot

### PR DESCRIPTION
**Pull request by dhuseby**
adds the bitrig cargo snapshot.  d/l the tarball from here: http://blog.linuxprogrammer.org/cargo-nightly-x86_64-unknown-bitrig.tar.gz

### CODE REVIEW

**Comment by alexcrichton** [comment]
Nice! I've uploaded the snapshot now, but I think that this will need to change the `dl-snapshot.py` script a bit as well. If you make the changes now I believe that a full and vanilla `./configure` + `make` should work just fine (with no modifications).

**Comment by dhuseby** [comment]
@alexcrichton done and done. :)

File: src/etc/dl-snapshot.py [hunk 1]
```diff
@@ -52,6 +52,9 @@
     elif arch == 'x86_64':
         me = win64
         new_triple = 'x86_64-pc-windows-gnu'
+elif target_os == 'bitrig':
+    me = bitrig64
```
Comment (alexcrichton) [inline src/etc/dl-snapshot.py]: I think this `bitrig64` variable needs to be defined above?

**Comment by dhuseby** [comment]
@alexcrichton yup, just fixed it. (oops)

**Pull request merged**